### DISCUSSION TITLE
Does pagination connection work on unions?

### DISCUSSION THREAD

**Discussion by BCantos17** [Netflix/dgs-framework#839] [Q&A] [answered]
Can I do something like

`directive @connection on UNION`

and then use that on either of the types to generation a connection from it?

like 

```
union Media @connection = Video | Image
```

If not is there an alternative to form of pagination for unions or a planned feature?

Right now when I attempt this on a union, I get an error
`The field type 'MediaConnection' is not present when resolving type Query`

**Comment by srinivasankavitha** [comment, marked as answer]
Hi - it is currently expected to work for objects and interfaces. We haven't added this for unions. Could you please open an issue for adding support for unions?

**Reply by BCantos17** [reply]
Thank you, opened a feature request https://github.com/Netflix/dgs-framework/issues/843
